docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.7"
+
+services:
+  pgdb:
+    build: env/postgres
+    image: pgdb
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_PASSWORD=<your_pg_password>
```

Comment:
Here is where the postgres password we setup on the first day is set for the container, so you'll just need to add your pw after the `=`, so it's something like `POSTGRES_PASSWORD=pw`.